src/charm.py

```diff
@@ -26,20 +28,89 @@ def __init__(self, *args: typing.Any) -> None:
         super().__init__(*args)
         self.framework.observe(self.on.config_changed, self.config_service)
 
+    def flask_container(self, require_connected: bool = True) -> Container:
+        """Get the flask application workload container controller.
+
+        Args:
+            require_connected: if set to ``True``, a runtime exception will be raised if the
```

Comment:
single quote?

Reply:
I think we are using the reStructuredText in docstrings? The reStructuredText use double back quotes for code blocks.